src/web_interface/templates/database/database_search.html

```diff
@@ -12,26 +14,26 @@
 <div class="row justify-content-center">
 
     <div class="col-xl-6 col-lg-8 col-md-10 mt-4">
+        <script type="text/javascript"
+                src="{{ url_for('static', filename='web_js/bootstrap-select.min.js') }}`"></script>
```

Comment:
This template imports `bootstrap-select.min.js` twice. Is there a reason for that?

Reply:
removed